kubectl-plugin/pkg/cmd/ray.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	cluster "github.com/ray-project/kuberay/kubectl-plugin/pkg/cmd/cluster"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	CLIName = "ray"
```

Comment:
do we use this anywhere?

Reply:
I was planning on using it for once we want to add the `version` command. Removed it since we currently dont have a `version` command.